src/components/PeopleTable.tsx

```diff
@@ -1,644 +1,86 @@
+import { FC } from 'react';
+import { Person } from '../types';
+import PersonItem from './PersonItem';
+import { Link, useSearchParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { getSearchWith } from '../utils/searchHelper';
+
 /* eslint-disable jsx-a11y/control-has-associated-label */
-export const PeopleTable = () => {
+type Props = {
+  people: Person[];
+};
+
+enum SortBy {
+  name = 'Name',
+  sex = 'Sex',
+  born = 'Born',
+  died = 'Died',
+}
+export const PeopleTable: FC<Props> = ({ people }) => {
+  const [searchParams] = useSearchParams();
+  const sort = searchParams.get('sort') || null;
+  const order = searchParams.get('order') || null;
+
+  const sortedPeople = [...people].sort((a, b) => {
+    if (sort === 'born' || sort === 'died') {
+      return order === 'desc' ? b[sort] - a[sort] : a[sort] - b[sort];
+    }
+
+    if (sort === 'name' || sort === 'sex') {
+      const comparison = a[sort].localeCompare(b[sort]);
+
+      return order === 'desc' ? comparison : -comparison;
```

Comment:
The comparison logic for string fields (`name` and `sex`) is incorrect. The current implementation reverses the comparison logic for ascending order. It should return `comparison` for ascending order and `-comparison` for descending order.